pro/eslint-plugin-pass-culture/lib/rules/use-date-fns-tz-format.js

```diff
@@ -30,6 +41,7 @@ module.exports = {
         'Please use the format function from date-fns-tz library instead of the one from date-fns. Please consider timezone in datetime formatting.',
       importIndividualFunctions:
         'Please import individual function from date-fns library. If format function is needed, please import it from date-fns-tz and take timezone into account.',
+      useTimezoneOption: 'Please use timezone option in format arguments.',
```

Comment:
Sur les messages peut-être parler du why non ?
Dans le genre "as some offers are in France DROM, you need to take TZ into account"